sorc/emcsfc_snow2mdl.fd/grib_utils.f

```diff
@@ -60,6 +87,7 @@ end subroutine grib_check
  SUBROUTINE SKGB2(LUGB,ISEEK,MSEEK,LSKIP,LGRIB,I1)
 !$$$  subprogram documentation block
```

Comment:
This subroutine documentation block needs to be moved about the subroutine. I know I have said this before...